cloudbuild.yaml

```diff
@@ -0,0 +1,15 @@
+steps:
```

Comment:
fyi, lws and jobset are slightly different: https://github.com/kubernetes-sigs/lws/blob/main/cloudbuild.yaml

not an expert on this file, so I am not sure if we need to do the same thing here.

Reply:
Yeah, they are using `make` as their build command. We are just using a simple `docker build` for now, so we are using a base image. I expect we will increase in sophistication as time goes on but this works for now I think. I've tested this on my personal GCP project's cloud build (I had to change the registry, but thats just permissioning issues)